src/components/DiffView/index.d.ts

```diff
@@ -0,0 +1,38 @@
+declare module 'react-diff-view' {
+  export interface HunkInfo {
```

Comment:
Note that we probably have no need to document the type for `HunkInfo` as it's never directly used by our code. We receive an array of `HunkInfo` objects from `parseDiff` and then pass them to the `Diff` component via a prop, but what they contain is irrelevant. We could possibly just use `type Hunks = Array<object>` instead of `type Hunks = Array<HunkInfo>` at line 12, below.

That's also the reason I documented `changes` as `Array<object>` here, rather than providing a concrete type for what a `change` is, because it's so deep that our code never really cares about it.